error: [Prusti: verification error] postcondition might not hold.
  --> $DIR/box.rs:6:11
   |
6  | #[ensures(!result)]
   |           ^^^^^^^
   |
note: the error originates here
  --> $DIR/box.rs:7:1
   |
7  | / fn box_test(x: i32) -> bool {
8  | | let y = Box::new(x);
9  | | let z = Box::new(y);
10 | | **z == 0
11 | | }
   | |_^
note: counterexample for "x"
   initial value: 0
   final value:   0
--> $DIR/box.rs:7:13
|
7  | fn box_test(x: i32) -> bool {
|             ^
note: counterexample for "y"
   final value:   box(0)
--> $DIR/box.rs:8:5
|
8  | let y = Box::new(x);
|     ^
note: counterexample for "z"
   final value:   box(box(0))
--> $DIR/box.rs:9:5
|
9  | let z = Box::new(y);
|     ^
note: counterexample for result
   final value:   true
--> $DIR/box.rs:7:24
|
7  | fn box_test(x: i32) -> bool {
|                        ^^^^

error: aborting due to previous error